project/AppDependencies.scala

```diff
@@ -2,17 +2,17 @@ import sbt._
 
 object AppDependencies {
 
-  lazy val bootstrapVersion = "8.2.0"
+  lazy val bootstrapVersion = "8.4.0"
 
   val compile = Seq(
     "uk.gov.hmrc"        %% "bootstrap-backend-play-30"   % bootstrapVersion,
-    "org.apache.axis2" % "axis2-kernel"               % "1.8.2"
+    "wsdl4j"              % "wsdl4j"                      % "1.6.3"
   )
 
   val test = Seq(
     "uk.gov.hmrc"            %% "bootstrap-test-play-30"      % bootstrapVersion % Test,
     "com.vladsch.flexmark"   % "flexmark-profile-pegdown"     % "0.64.8"         % "test, it",
```

Comment:
We don't need `flexmark`, `scalatestplus-play` or `wiremock` because they're all provided by `bootstrap-test-play-30`.

Reply:
I've made the change, I tried it out last time but my test scope was failing for AT. So I extended the test scope for bootstrap-test-play-30 to the integration tests, seems like we are good. Thanks man!